cmd/buildctl/build.go

```diff
@@ -315,7 +316,16 @@ func buildAction(clicontext *cli.Context) error {
 }
 
 func writeMetadataFile(filename string, exporterResponse map[string]string) error {
-	b, err := json.Marshal(exporterResponse)
+	out := make(map[string]interface{})
+	for k, v := range exporterResponse {
+		dt, err := base64.StdEncoding.DecodeString(v)
+		if err != nil {
+			out[k] = v
+			continue
+		}
+		out[k] = json.RawMessage(dt)
```

Comment:
Still need to check that it is a valid JSON.

Reply:
Was wondering if it was necessary actually as we do that on known b64 encoded keys in our response but yes decoded data could be trash.